src/testing/Home.test.js

```diff
@@ -0,0 +1,11 @@
+import '@testing-library/jest-dom';
```

Comment:
- [ ] You have done well adding tests for simulating user actions using the testing library, but you are still missing snapshot tests for all your components. Kindly add snapshot tests for the components captured below.

  Pages folder | Components folder
  :---:|:---:
  ![rririw](https://user-images.githubusercontent.com/81468675/236201778-c46d6130-7f8e-45d3-a346-ac804892a848.jpg) | ![duirir](https://user-images.githubusercontent.com/81468675/236201786-d9df94f5-4beb-4cb9-af62-37294455747d.jpg)